templates/plugin/dev/tsconfig.json

```diff
@@ -2,6 +2,10 @@
   "extends": "../tsconfig.json",
   "exclude": [],
   "include": [
+    "**/*.js",
+    "**/*.jsx",
+    "**/*.mjs",
+    "**/*.cjs",
```

Comment:
There is a `.js` file in the dev folder that throws an error for not being in the project